rest/db-file-get.rst

```diff
@@ -19,7 +38,10 @@ respectively.
       "local": { /* a file entry */ }
     }
 
-A file entry looks like this::
+``local`` and ``global`` refer to the current file on the local device and the
+globally newest file, respectively.  A file entry looks like this:
```

Comment:
There is a factual error here.  The `mtime` attribute is not part of the respective file entry, but top-level next to `availability`.  When that is fixed, the outer object wrapping can go away in this code sample.